src/components/Admin/Admin.js

```diff
@@ -33,16 +30,19 @@ export default function Admin() {
       if (!text || text == '') return;
 
       const result = await checkHandle(text);
+      setDisplay(result);
 
-      updateDisplay(result);
       let lastHistory = await getHistory();
-      updateHistory(lastHistory);
-      //setInputValue('');
+      setHistory(lastHistory);
     } catch (error) {
       console.error('Error on adding link to db');
     }
   };
 
+  const handleNewCheck = async () => {
+    if (inputValue) window.location.replace(`/handle/${inputValue}`);
+  };
+
   const handleChange = ({ which, target, keyCode }) => {
     if (which === 13 || keyCode === 13) {
```

Comment:
Will it be possible to assign the `submit` role to the button and capture Enter this way rather than have both `onChange` and `onKeyDown` on the control?